### PR TITLE
Remove permission requests at onboarding

### PR DESCRIPTION
**Pull request by carlonzo**
Storage permission is request when selecting another storage path for the download

### CODE REVIEW

File: mobile/src/main/java/butter/droid/ui/preferences/PreferencesActivity.java [hunk 1]
```diff
@@ -218,6 +243,18 @@ public void openChangelog() {
         startActivity(AboutActivity.getIntent(this));
     }
 
+    @Override
+    public void onRequestPermissionsResult(int requestCode, @NonNull String[] permissions, @NonNull int[] grantResults) {
+        super.onRequestPermissionsResult(requestCode, permissions, grantResults);
+
+        if (requestCode == OPEN_DIRECTORY_SELECTOR_PERMISSION){
+            if (grantResults[0] == PackageManager.PERMISSION_GRANTED){
+                openDirectorySelector(Prefs.STORAGE_LOCATION, R.string.storage_location, preferencesHandler.getStorageLocation());
```
Comment (carlonzo) [inline mobile/src/main/java/butter/droid/ui/preferences/PreferencesActivity.java]: this is an horrible solution. here I had to copy-paste the parameters from the `BasePreferencesPresenter`

**Comment by blazsolar** [comment]
There are some checkstyle violations in PreferencesActivity. Could you fix that.

**Comment by carlonzo** [comment]
Yep!

**Comment by blazsolar** [comment]
Already in develop with some minor changes.

**Pull request closed without merging**